tests/robot-tests/scripts/get_auth_tokens.py

```diff
@@ -144,7 +142,7 @@ def get_identity_info(
     chrome_options.add_argument("--ignore-certificate-errors")
     chrome_options.add_argument("--disable-logging")
     chrome_options.add_argument("--log-level=\3")
-    driver = webdriver.Chrome(options=chrome_options, desired_capabilities=DesiredCapabilities.CHROME)
```

Comment:
Why has this been removed?

Reply:
Desired capabilities is no longer a thing in Selenium. Things it previously configured can be changed via the options object:
https://support.smartbear.com/crossbrowsertesting/docs/automated-testing/frameworks/selenium/about/use-of-desiredcapabilities-has-been-deprecated.html